graphql_api/types/repository/repository.py

```diff
@@ -554,6 +554,8 @@ async def resolve_test_results(
         repoid=repository.repoid,
         branch=filters.get("branch") if filters else None,
         parameter=generate_test_results_param,
+        testsuites=filters.get("testsuites") if filters else None,
```

Comment:
test_suites

Frontend will likely name the variable testSuites which will get converted to test_suites due to https://github.com/codecov/codecov-api/blob/9aa6458ad023fe0ec55993ad9134bbd0af5fb3f7/graphql_api/schema.py#L7